mantidproject/mantid
# Add support for Wildes polarization correction

**Description of work.**

- Add support in `ReflectometryReductionOneAuto` for Wildes polarization correction
- Workspaces are removed from the input group first and passed as a list. The Wildes algorithm creates new workspaces in the output group and we then delete the original workspaces. This gets around a problem where the Wildes algorithm doesn't support input workspaces in a group, particularly if the output group name is the same as the group the input workspaces are in.
- Copy history from the input workspaces in the Wildes algorithm.

Fixes #23082

**Report to:** Andrew at ISIS

**To test:**
- Save these files and run the `test.py` script: [23082_enable_wildes_in_RROA.zip](https://github.com/mantidproject/mantid/files/2236127/23082_enable_wildes_in_RROA.zip)
- Just check that the script completes without errors. It runs several reductions but they are not intended to be real reductions; they just exercise various combinations of methods and options. I'll get the scientists to validate the results.

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
